stable/ghost/templates/deployment.yaml

```diff
@@ -17,7 +17,8 @@ spec:
     spec:
       initContainers:
       - name: volume-permissions
-        image: busybox
+        image: "{{ .Values.initContainerImage }}"
+        imagePullPolicy: {{ default "" .Values.imagePullPolicy | quote }}
```

Comment:
ok I misunderstood your request. it should be alright to have this structure for the newly added initContainer /cc @tompizmor